apps/cli/src/commands/environment/list.environment.ts

```diff
@@ -24,9 +25,39 @@ export class ListEnvironment extends BaseCommand {
     ]
   }
 
-  async action({ args }: CommandActionData): Promise<void> {
+  getOptions(): CommandOption[] {
+    return [
+      {
+        short: '-p',
+        long: '--page <page>',
+        description: 'Index of the page.'
+      },
+      {
+        short: '-l',
+        long: '--limit <items>',
+        description: 'Total number of items per page.'
+      },
+      {
+        short: '-o',
+        long: '--order <sort order>',
+        description:
+          'Order to sort by - either ascending (ASC) or descending (DESC).'
+      },
+      {
+        short: '--sort',
+        long: '--sort <field name>',
+        description: 'Field to sort by.'
+      },
+      {
+        short: '-s',
+        long: '--search <search>',
+        description: 'Search term.'
+      }
+    ]
+  }
+  async action({ args, options }: CommandActionData): Promise<void> {
     const [projectSlug] = args
-
+    const { page, limit, order, sort, search } = options
```

Comment:
I think you can just disregard this line, and use `...options` while making the http call.